src/UCommerce.SiteCore.Installer/Steps/InstallStep.cs

```diff
@@ -14,108 +15,107 @@ public InstallStep(string connectionString, string sitecorePath, bool backupDb,
-           //      new EnableSitecoreCompatibilityApp(sitecoreVersionChecker, sitecoreInstallerLoggingService));
-           //
-           //  // Remove CatalogSearch widget
-           //  steps.Add(
-           //      new DeleteDirectory($"{virtualAppsPath}/Widgets/CatalogSearch"));
-           //  steps.Add(
-           //      new DeleteDirectory($"{virtualAppsPath}/Widgets/CatalogSearch.disabled"));
-           //
-           //  // Enable Sanitization app
-           //  steps.Add(new MoveDirectory($"{virtualAppsPath}/Sanitization.disabled", $"{virtualAppsPath}/Sanitization", true));
-           //  steps.Add(new DeleteFile($"{virtualAppsPath}/Sanitization/bin/AngleSharp.dll"));
-           //  steps.Add(new DeleteFile($"{virtualAppsPath}/Sanitization/bin/HtmlSanitizer.dll"));
-           //
-           //  //Clean up unused configuration since payment integration has move to apps
-           //  steps.Add(
-           //      new DeleteFile("~/sitecore modules/shell/ucommerce/Configuration/Payments.config"));
-           //
-           //  steps.Add(new MoveUcommerceBinaries());
-           //  steps.Add(new MoveResourceFiles());
-           //
-           //  ComposeConfiguration();
-           //  ComposePipelineConfiguration();
-           //  steps.Add(
-           //      new RenameConfigDefaultFilesToConfigFilesStep("~/sitecore modules/Shell/uCommerce/Apps", false));
-           //  steps.Add(new MoveDirectoryIfTargetExist(
-           //      $"{virtualAppsPath}/SimpleInventory.disabled",
-           //      $"{virtualAppsPath}/SimpleInventory"));
-           //  steps.Add(new MoveDirectoryIfTargetExist(
-           //      $"{virtualAppsPath}/Acquire and Cancel Payments.disabled",
-           //      $"{virtualAppsPath}/Acquire and Cancel Payments"));
-           // // Set up search providers
-           //  ToggleActiveSearchProvider(virtualAppsPath);
-           //
-           //  // Clean lucene from disk
-           //  SearchProviderCleanup("~/sitecore modules/Shell/uCommerce/Apps");
-           //
-           //  //Create back up and remove old files
-           //  RemovedRenamedPipelines();
-           //
-           //  steps.Add(new CreateApplicationShortcuts());
-           //  steps.Add(new CreateSpeakApplicationIfSupported(sitecoreVersionChecker));
-           //
-           //  // Move sitecore config includes into the right path
-           //  ComposeMoveSitecoreConfigIncludes(sitecoreVersionChecker);
-           //  
-           //  // Clean up System.Collections.Immutable.dll in Lucene App since it is no longer used
-           //  steps.Add(new DeleteFile($"{virtualAppsPath}/Ucommerce.Search.Lucene/bin/System.Collections.Immutable.dll"));
-           //  steps.Add(new DeleteFile($"{virtualAppsPath}/Ucommerce.Search.Lucene.disabled/bin/System.Collections.Immutable.dll"));
-           //
+            steps.Add(new DeleteFile(new FileInfo("C:\\code\\hej.txt"), new SitecoreInstallerLoggingService()));
```

Comment:
Should this not be cleaned up? I assume this was used for testing

Reply:
Good point